hathor/event/storage/rocksdb_storage.py

```diff
@@ -91,7 +91,7 @@ def get_last_event(self) -> Optional[BaseEvent]:
     def get_last_group_id(self) -> Optional[int]:
         return self._last_group_id
 
-    def clear_events(self) -> None:
+    def reset_events(self) -> None:
         self._last_event = None
```

Comment:
Should you reset these parameters in RocksDB as well?

Reply:
`self._last_event` is not stored explicitly in RocksDB, it's calculated dynamically from the events column family, and then stored on memory in `self._last_event`

`self._last_group_id` is stored explicitly in RocksDB, so it's deleted from there: `self._db.delete((self._cf_meta, _KEY_LAST_GROUP_ID))`

Reply:
Got it. Thanks!